apps/system/js/app_window.js

```diff
@@ -1196,7 +1196,8 @@
         case 'application-name':
           // Apps have a compulsory name field in their manifest
           // which takes precedence.
-          if (this.manifestURL || this.webManifestURL) {
+          var emptyString = !(detail.content.trim());
```

Comment:
Are we sure detail.content is always defined for the metachanged event? I'm just thinking, what if someone had:
<meta name="application-name">
Do we care? Do I care? I don't know the answer.